examples/index.php

```diff
@@ -39,8 +39,8 @@
 
 $sessionMiddleware = new SessionMiddleware(
     new Sha256(),
-    'a very complex symmetric key',
-    'a very complex symmetric key',
+    'c9UA8QKLSmDEn4DhNeJIad/4JugZd/HvrjyKrS0jOes=', // signature key (generated using a CSPRNG)
+    '6TL/lft85cQNibsl/IG++eh179F7CJOsql7V9rcOSsk=', // verification key (generated using a CSPRNG)
```

Comment:
@AndrewCarterUK this example enforces `withSecure(false)` (the library works only with HTTPS otherwise - by design!). If you look at the `fromSymmetricKeyDefaults`, this will be more clear :-)